systemd/manager.go

```diff
@@ -130,10 +130,25 @@ func (m *systemdUnitManager) TriggerStart(name string) {
 	}
 }
 
-// TriggerStop asynchronously starts the unit identified by the given name.
+// Stop stops the unit identified by the given name.
+// This function blocks until the underlying unit is actually stopped.
+func (m *systemdUnitManager) Stop(name string) {
+	retChan := make(chan string)
+	m.stopUnit(name, retChan)
+
+	jobReply := <-retChan
```

Comment:
Then where is the block happening? I would assume that `stopUnit()` is quick because it does stuff in a goroutine / async and this channel is written to, when the stop finished. So the read here is blocking / slow. So time should be meaning full.